bstoudt01/BrewMore
# Bh

# Description : 
Add Brewhouse Component to App. User can now add instances of any brand to a list to see a total of required ingredients needed to complete the brews. 

## Type of change: 

# Testing Instructions
Login and add one or more brands to brands list. go to brewhouse component and click add on the brand you want to brew. 
see the grains listed on the bottom of the page, and a new brew card appears.
click the "+" on the brew card to add another brew instance to that brand, or "-" to remove an instance.

# Checklist:
